src/components/MenuCategories/MenuCategories.tsx

```diff
@@ -1,36 +1,57 @@
 import { CategoryDto } from 'dtos/CategoryDto';
-import { Dispatch, ReactElement, SetStateAction } from 'react';
+import { Dispatch, ReactElement, SetStateAction, useState } from 'react';
 import { Button } from 'components/Button/Button';
+import { faArrowRight, faArrowLeft } from '@fortawesome/free-solid-svg-icons';
 import cn from 'classnames';
+import { motion } from 'framer-motion';
+import { ArrowToggle } from 'components/Icons/ArrowToggle';
+
 interface Props {
   setSelectedCategory: Dispatch<SetStateAction<number>>;
   selectedCategory: number;
   categories: CategoryDto[];
 }
 
 export const MenuCategories = ({ categories, setSelectedCategory, selectedCategory }: Props): ReactElement => {
+  const [index, setIndex] = useState(0);
+
+  const handleNext = (): void => {
+    setIndex((prevIndex) => (prevIndex + 5 >= categories.length ? 0 : prevIndex + 5));
+  };
+
+  const handlePrev = (): void => {
+    setIndex((prevIndex) => (prevIndex - 5 < 0 ? prevIndex : prevIndex - 5));
+  };
+
   return (
     <div className="mt-5 flex flex-col items-center">
       <p className="text-[36px] font-[300]">Menu</p>
       <div className="mt-5 w-[95%] divide-y-2 divide-slate-700">
-        <div>
-          {categories.map(
+        <div className="group flex items-center justify-between">
+          <ArrowToggle icon={faArrowLeft} handleIndex={handlePrev} />
```

Comment:
a nitpick but I'd rename `handleIndex` to `onClick` to keep it consistent to other components